### PR TITLE
Add some python packages

### PR DESCRIPTION
**Pull request by mitchty**
Add in a few python pypi packages:
- requests-cache
- howdoi
- nose-parameterized
- jdatetime
- dateparser

### CODE REVIEW

**Comment by mitchty** [comment]
Needed a few of these for testing a few packages last week. Well except howdoi which just lets me be lazy at searching. I think I got the licenses right, not great at knowing the difference between bsd3/2 and so on.

Ping @domenkozar as per irc. Can amend anything I got wrong if needed.

**Comment by mitchty** [comment]
Oops, https://travis-ci.org/NixOS/nixpkgs/builds/92604936#L153 silly mistake, will fix and update.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3266,6 +3266,100 @@ let
...
+    name = "requests-cache-${version}";
+    version = "0.4.10";
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "http://pypi.python.org/packages/source/r/requests-cache/${name}.tar.gz";
+      sha256 = "671969d00719fa3e80476b128dc9232025926884d0110d4d235abdd9c3508fc0";
+    };
+
+    propagatedBuildInputs = with self; [ self.six requests ];
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: we probably want `requests2`

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -3266,6 +3266,100 @@ let
...
+    name = "dateparser-${version}";
+    version = "0.3.1";
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "http://pypi.python.org/packages/source/d/dateparser/${name}.tar.gz";
+      sha256 = "56c291a45398e9172d53201ac213226989295749191c1f02d8f3b593b6f88e48";
+    };
+
+    propagatedBuildInputs = with self; [ self.six jdatetime pyyaml dateutil nose nose-parameterized mock ];
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: nose and mock should be in `buildInputs` since they're build time dependencies

**Comment by jagajaga** [comment]
Please name commits following our guide in `CONTRIBUTING.md`.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -3266,6 +3266,102 @@ let
...
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "http://pypi.python.org/packages/source/r/requests-cache/${name}.tar.gz";
+      sha256 = "671969d00719fa3e80476b128dc9232025926884d0110d4d235abdd9c3508fc0";
+    };
+
+    buildInputs = with self; [ mock sqlite3 ];
+
+    propagatedBuildInputs = with self; [ self.six requests2 ];
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: No need to specify `self.` twice

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -3266,6 +3266,102 @@ let
...
+  howdoi = buildPythonPackage (rec {
+    name = "howdoi-${version}";
+    version = "1.1.7";
+
+    src = pkgs.fetchurl {
+      url = "http://pypi.python.org/packages/source/h/howdoi/${name}.tar.gz";
+      sha256 = "df4e49a219872324875d588e7699a1a82174a267e8487505e86bfcb180aea9b7";
+    };
+
+    propagatedBuildInputs = with self; [ self.six requests-cache pygments pyquery ];
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: same here

**Pull request merged**